### DISCUSSION TITLE
How to get the PDF of an invoice

### DISCUSSION THREAD

**Discussion by Leonardopergola** [fattureincloud/api#475] [Q&A] [answered]
There is any method to get the PDF of an issued document? 
I'm building a tool with zapier and fatture in cloud. I managed to generate the invoices automatically. Now i would like to retrieve the PDF of the issued document and save it on a google drive shared with the client. 
Does anybody have experience with something similar?
Thank you in advance

**Comment by emajo** [comment, marked as answer]
Hi @Leonardopergola, I think after the creation step you could add a _Google Drive Upload file_ action and set the File field as the attachment url you receive from the creation step.

That should be enough,
thanks,
Emanuele.

**Reply by valmoz** [reply]
Hi @Leonardopergola were you able to solve this issue?

**Reply by Leonardopergola** [reply]
Yes! Thank you very much. Using the field data.url
